src/Toolkit/Toolkit/UI/Controls/BasemapGallery/BasemapGalleryController.cs

```diff
@@ -324,20 +324,10 @@ private static async Task<IList<BasemapGalleryItem>> PopulateFromDefaultList(Can
                 listOfBasemaps.Add(new BasemapGalleryItem(basemap));
             }
 
-#if !WINDOWS_XAML && !NETCOREAPP
-            await Task.WhenAll(listOfBasemaps.Select(gi => gi.LoadAsync()));
-#else
             foreach (var item in listOfBasemaps)
             {
-                try
-                {
-                    await item.LoadAsync();
-                }
-                catch (Exception)
-                {
-                }
+                _ = item.LoadAsync();
```

Comment:
Does this create risk of unobserved exceptions if one of the items fail to load?

Reply:
Nope: https://github.com/Esri/arcgis-maps-sdk-dotnet-toolkit/blob/088a15dae871581d0ccf5ace95f89ec506431c3c/src/Toolkit/Toolkit/UI/Controls/BasemapGallery/BasemapGalleryItem.cs#L90